<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TitleBar</class>
 <widget class="QWidget" name="TitleBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>58</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#TitleBar {
	background-color: #212121;
}

#Title {
	color: white;
	margin-left: 10px;
}

#TitleIcon {
	border-image: url(:/favicon/icon_64x64.png)
}

#BtnClose { 
	border-image: url(:/TitleBar/img/cls_btn.svg)
}
#BtnMax { 
	border-image: url(:/TitleBar/img/max_btn.svg)
 }
#BtnMin { 
	border-image: url(:/TitleBar/img/min_btn.svg);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="TitleIcon">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="Title">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string>Click Mapper</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="Spacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>918</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="TitleBarBtns">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="BtnMin">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnMax">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnClose">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../src/src.qrc"/>
 </resources>
 <connections/>
</ui>
